### PR TITLE
Feature/mob 4398 quote list snapshot tests

### PR DESCRIPTION
**Pull request by AleksanderWedrychowskiKarhoo**
## JIRA
[MOB-4398](https://karhoo.atlassian.net/browse/MOB-4398)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [ ] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: KarhooUISDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2491,6 +2502,30 @@
 			name = "Recovered References";
 			sourceTree = "<group>";
 		};
+		519D100528FDBF8400EF0EB0 /* QuoteList */ = {
+			isa = PBXGroup;
+			children = (
+				519D100728FDBF9500EF0EB0 /* Fitler */,
```
Comment (dianakarh) [inline KarhooUISDK.xcodeproj/project.pbxproj:2508]: Looks like we have a type in the name

File: KarhooUISDK/API/KarhooComponents.swift [hunk 2]
```diff
@@ -21,8 +21,8 @@ public class KarhooComponents: BookingScreenComponents {
             cornerRadious: UIConstants.CornerRadius.large,
             borderLine: true,
             dropShadow: false,
-            verticalPadding: 5.0,
-            horizontalPadding: 5.0,
+            verticalPadding: 0,
+            horizontalPadding: 0,
```
Comment (dianakarh) [inline KarhooUISDK/API/KarhooComponents.swift:25]: Why the change in padding?

File: KarhooUISDKTests/Mocks/KarhooSDK/MockQuote.swift [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import KarhooSDK
+
+extension Quote {
+    static func mock(quoteType: QuoteType = .estimated, vehicleType: String = "standard") -> Quote {
+        Quote(
+            id: UUID().uuidString,
```
Comment (dianakarh) [inline KarhooUISDKTests/Mocks/KarhooSDK/MockQuote.swift:15]: There should be a helper method to get a random ID somewhere in the code. Can we reuse that?

File: KarhooUISDKTests/Mocks/KarhooSDK/MockQuote.swift [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            quoteType: quoteType,
+            source: .fleet,
+            pickUpType: .default,
+            fleet: .init(id: UUID().uuidString, name: "Mocked Fleet"),
```
Comment (dianakarh) [inline KarhooUISDKTests/Mocks/KarhooSDK/MockQuote.swift:19]: same as for the quote id

File: KarhooUISDK/API/KarhooComponents.swift (hunk 2 above)
Comment (AleksanderWedrychowskiKarhoo) [reply KarhooUISDK/API/KarhooComponents.swift:25]: 5.0 margin resulted in misalignment in QuoteList. There is no reason to use internal padding - if design require so, we or DP can use external layout to handle that.

**Comment by AleksanderWedrychowskiKarhoo** [comment]
PR closed due to multiple changes done in next PR
